packages/amplify-util-mock/src/func/index.ts

```diff
@@ -45,7 +47,7 @@ export async function start(context: $TSContext) {
   const invoker = await getInvoker(context, { resourceName, handler: lambdaConfig.handler, envVars: lambdaConfig.environment });
   context.print.blue('Starting execution...');
   try {
-    const result = await timeConstrainedInvoker(invoker({ event }), context.input.options);
+    const result = await timeConstrainedInvoker(invoker({ event }), context.input.options as $TSAny);
```

Comment:
this was typed as `$TSAny` before on the CLI side, so I'm casting this for simplicty, the target type is anyway one optional property.
@sdstolworthy we should dry-run context changes in data repo through tag release going forward.